wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,201 @@
+![](img/sample-import.jpg "Sample import window")
 
-**Right-section** is **settings for the selected sound file.**
+The left side of the sample import window lists all audio samples from the beatmap folder. They can be applied to selected objects using settings on right side of the window.
 
-##### Basics
+##### Basic
 
 | Name | Description |
 | :-- | :-- |
-| CustomSet | **Ignoring** selected custom hitsound in this editor, and apply the **default** hitsound **SET** and volume without using timing point. ( Use basic sample must be enable in order to use this to be able to work as its intended )ex : added soft-hitwhistle77.wav with 90% volume.Note that you must add the whistle to the note manually as well. **Also this feature is not recommended while the normal APPLY function in this editor is easier to be used than this one while both of this basically works the same.** |
-| Volume | How loud you want this sound file be. Use integer value only. ( 8 - 100 ) |
-| Use basic sample | **Ignoring** selected custom hitsound in this editor , and apply only the volume on selected notes.This means , no matter what the notes is about, this feature only changes the selected notes volume.Ex : this can be used to change multiple volume on different notes with different hitsound sample at once or just enhance the default hitsound ignoring any timing point's volume setting. |
+| `CustomSet` | Apply default hitsound set and volume without using these of the current timing point. Ignores the left-side sample selection. *Note: For this action to work properly, `Use basic sample` must be enabled.* |
+| `Volume` | How loud this sound file should be. Use only integer values from 8 to 100. |
+| `Use basic sample` | **Ignores** selected custom hitsound in this editor, and applies only the volume on selected notes. This means, no matter what the notes are about, this feature only changes the selected notes volume. |
 
 ##### Buttons
 
 | Name | Description |
 | :-- | :-- |
-| Play | Play the selected sound file |
-| Import | Import the custom sound file from a subfolder inside the beatmap's folder |
-| Delete | **Delete the custom sound file.** To remove the sound file associated with the note, use the "Reset" button. |
-| Apply | Apply this selected custom sound file to this note. |
-| Sample | Put this selected custom sound file to the storyboard with respect to current timestamp. |
-| Sample list | Storyboarded sound files. Refer "Sample event list" below |
-| Reset | Remove the sound file associated with the note |
-| Cancel | Close the window. |
-
-#### Sample event list
+| `Play` | Play selected sample. |
+| `Import` | Add more samples to the beatmap folder from other locations. |
+| `Delete` | Delete the selected sample's file, **irreversibly**. |
+| `Apply` | Apply selected sample to the selected note. |
+| `Sample` | Add selected sample as a storyboard event at the current timestamp. |
```

Comment:
til this was the way you were supposed to add storyboard samples

```suggestion
| `Sample` | Add selected sample as a [storyboard audio event](/wiki/Storyboard/Scripting/Audio) at the current timestamp. |
```

Reply:
it's one of the ways — the old good "write it in .osb" existed well before osu!mania has been introduced